.github/label-pr-config.yml

```diff
@@ -0,0 +1,84 @@
+autolabeler:
+  - label: 'Type: documentation'
+    files:
+      - '*.md'
+    # Checks there is no word character (alphanumeric & underscore) before target words, or after to avoid accidental matches
+    title:
+      - '/(?<![\w])((doc)|(docs)|(documentation))(?![\w])/i'
+  - label: 'Type: fix'
+    branch:
+      - '/fix\/.+/'
+    title:
+      - '/(?<![\w])((fix)|(fixes))(?![\w])/i'
```

Comment:
should also look for `fixed` and `bug` words in title regex. Also if possible lets search for substring instead of entire string match. So we dont need to add different modifications of same word. for ex if we just look for substring fix it would work when we have fixed, fixes.

Reply:
That grouping for the fix category makes sense! The regex has been updated to include those words as well. I was thinking of using exact word matches with whitespace boundary checks instead of substring to prevent accidental matching causing labels to be applied to unintended contexts.
Ex ([words containing these letters](https://wordfinder.yourdictionary.com/words-with-the-letter/) - in the contains field):  
`bug` -> `debug`, `buggy`, `debugging`
`fix` -> `prefix`, `affix`, `postfix`, `suffix`, etc.

If listing all the words is too cumbersome, what do you think about creating mutually exclusive word ending regex options (next example is not actual regex, but conceptual example)?
fix/es/ed -> matches fix, fixes, fixed

Reply:
@timkimadobe I like the idea. Alternatively instead of substring we can pair your idea with `starts with` string. starts with fix, bug etc.

Reply:
Ok! With the starts with strategy, were you thinking of only applying the label if the word is at the start of the title?
Ex for matching cases: 
"Fix for some feature"
"Fixes broken link issue"

Ex: for non-matching cases:
"Feature fix for ..."
"Redirect issue fixed"

Reply:
I was saying starts with for a word. FIXed, FIXes etc.. which will handle the false positive cases you mentioned like debug, prefix etc..

Reply:
Got it, that makes sense! Thanks for clarifying. I'll update the regex pattern to be able to handle the word suffix patterns we're looking for